File: learn-academy/technical-interviews/2022/delta/prompt.js
// Stark Intelligence is building a new feature for Tortuga. 
// The AI will listen for a phrase and say it back in reverse.

// Expected input: I love apples
// Expected output: apples love I

const input = 'I love apples'

const backwords = (phrase) => {
  const phraseArray = phrase.split(' ')
  const reversedPhraseArray = phraseArray.reverse()
  let backwords = reversedPhraseArray.join(' ')
  return backwords
}

console.log(backwords(input))

File: learn-academy/technical-interviews/2022/delta/prompt.rb
# Stark Intelligence is building a new feature for Tortuga. 
# The AI will listen for a phrase and say it back in reverse.

# Expected input: I love apples
# Expected output: apples love I

input = 'I love apples'

def backwords(phrase)
  phraseArray = phrase.split(' ')
  reversedPhraseArray = phraseArray.reverse
  reversedPhraseArray.join(' ')
end

p backwords(input)